FinniversKit/Sources/DNA/Color/ColorProvider.swift

```diff
@@ -129,13 +189,13 @@ public struct DefaultColorProvider: ColorProvider {
     public var textDisabled: UIColor {
         .dynamicColor(defaultColor: .blueGray300, darkModeColor: .darkSardine)
     }
-    
+
     public var textAlert: UIColor {
         .lightNuttyBrown
     }
 
     public var textCritical: UIColor {
-        .dynamicColor(defaultColor: .red600, darkModeColor: .red400)
+        .dynamicColor( defaultColor: .red600, darkModeColor: .red400 )
```

Comment:
No space before/after parenthesis.

Reply:
I don't know what you're talking about, there's no space there 😉 

(this would be way more convincing if you couldn't see all my changes 🤷 )